pkgs/top-level/python-packages.nix

```diff
@@ -1583,6 +1583,8 @@ in {
 
   bunch = callPackage ../development/python-modules/bunch { };
 
+  bundlewrap = callPackage ../development/python-modules/bundlewrap { };
```

Comment:
If this is only a cli tool we do not need or want this entry.

Reply:
It is mainly intended for CLI use, but I can imagine people bypassing the CLI and directly hooking into bundlewrap from their CD scripts.